business/checkers/k8sgateway_checker.go

```diff
@@ -0,0 +1,48 @@
+package checkers
+
+import (
+	k8s_networking_v1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
+
+	"github.com/kiali/kiali/business/checkers/k8sgateways"
+	"github.com/kiali/kiali/models"
+)
+
+const K8sGatewayCheckerType = "k8sgateway"
+
+type K8sGatewayChecker struct {
+	K8sGateways           []*k8s_networking_v1alpha2.Gateway
+	WorkloadsPerNamespace map[string]models.WorkloadList
+	IsGatewayToNamespace  bool
```

Comment:
Are these 2 variables used? IsGatewayToNamespace and WorkloadsPerNamespace

Reply:
Those are not used, I will remove them, thanks.